kubernetes/loculus/values.yaml

```diff
@@ -96,6 +96,16 @@ defaultOrganismConfig: &defaultOrganismConfig
         noInput: true
         perSegment: true
         oneHeader: true
+      - name: latitude
+        displayName: Latitude
+        header: Sample details
+        type: float
+        guidance: Decimal degree format of latitude, i.e. values in range -90 to 90.
```

Comment:
Out of interest: Latitude and longitude of what?

Reply:
Geocoordinates